reana/utils.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA utils."""
```

Comment:
"REANA cluster component utilities." may be more precise so that end users don't think these are for them... Perhaps we could even name the file more specifically than just `utils.py`, something like `k8s_utils.py` or such...